src/Lykke.Service.ChainalysisProxy/Modules/ServiceModule.cs

```diff
@@ -46,9 +48,20 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<ShutdownManager>()
                 .As<IShutdownManager>();
 
-            // TODO: Add your dependencies here
+            var proxyUserRepository = new ChainalysisProxyUserRepository(
+                AzureTableStorage<ProxyUser>.Create(_dbSettings.ConnectionString(x => x.DataConnString),
+                    "ProxyUser", _log));
+            builder.RegisterInstance<IChainalysisProxyUserRepository>(proxyUserRepository).SingleInstance();
```

Comment:
use the Register method. It create an instance only when some one need it.
The same for the below method